pkg/util/kubernetes/autoscalers/processor.go

```diff
@@ -190,6 +192,10 @@ func (p *Processor) validateExternalMetric(emList map[string]custommetrics.Exter
 		}
 	}
 	log.Debugf("Processed %d chunks", len(chunks))
+
+	if err := p.updateRateLimitingMetrics(); err != nil {
```

Comment:
quick question: since now we query can be executed in //, is is possible that the values set in the metrics are not the last value send by the server? 
Do you think it can be a problem?

Reply:
all queries are indeed made in parallel but we are after the wait so we read from the client the last value stored.
Then, because the backend sets the rate limit stats for each call, not in parallel, the last value stored should be the most up to date one